_config/config.yml

```diff
@@ -6,4 +6,7 @@ After:
 ---
 Controller:
   extensions:
-    - QuickFeedbackExtension
\ No newline at end of file
+    - QuickFeedbackExtension
+
+QuickFeedbackExtension:
```

Comment:
Perhaps this could be a default in the php class private static anyway and then a user can overide (Have had issues with the way yaml config overrides if there are multiple duplicate directives being set.